archpkg/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+pkgname=d1to2fix
+pkgver=HEAD
+pkgrel=1
+url="https://github.com/sociomantic-tsunami/d1to2fix"
+pkgdesc="D1 to D2 conversion helper tool"
+license=("custom")
+depends=('gcc-libs')
+source=("$pkgname.tar.xz")
+sha256sums=('SKIP')
+arch=("any")
+makedepends=('git' 'dmd' 'gcc' 'dtools')
+options=(!buildflags)
+
+pkgver()
+{
+    cd $srcdir/$pkgname
+    git describe --long | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
+}
+
+build()
+{
+    cd $srcdir/$pkgname
+    git submodule update --init
+    make all F=production
+}
+
+package()
+{
+    mkdir -p $pkgdir/usr/bin/
+    install -m 755 $srcdir/$pkgname/build/production/bin/$pkgname $pkgdir/usr/bin/
+    mkdir -p $pkgdir/usr/share/licenses/$pkgname
+    install -m 644 $srcdir/$pkgname/LICENSE $pkgdir/usr/share/licenses/$pkgname
```

Comment:
I feel dumb for not thinking about it before. `fpm -t pacman` will probably work (will try right now) - though that makes situation even worse because you will have to decide on names and flags :)